src/geometry/euclidean.lean

```diff
@@ -499,50 +490,50 @@ Euclidean affine spaces.
 -/
 open add_torsor inner_product_geometry
 
```

Comment:
This is a place where an `open` of `add_torsor` is left unchanged. Unless they're actually using fields of the class definitions directly, I'd expect that all `open`s of `add_action` or `add_torsor` (or `affine_space`) can go away with the removal of those namespaces.

Reply:
```suggestion
-/
open inner_product_geometry

```